.github/workflows/release.yml

```diff
@@ -30,10 +30,6 @@ jobs:
           identity-provider: ${{ secrets.IDENTITY_PROVIDER }}
           service-account-email: ${{ secrets.SERVICE_ACCOUNT_EMAIL }}
           project-id: ${{ secrets.PROJECT_ID }}
-          prebuild-command: |
-            cd functions
-            pnpm install
-            cd ..
```

Comment:
I was only referencing the `cd ..` line, not the two lines above. I believe that those are still required, no?

Reply:
oh. ok. I won't know until it runs.